CanvasCore/CanvasCore/Quizzes/Quiz Details/QuizDetailsViewController.swift

```diff
@@ -66,6 +66,7 @@ class QuizDetailsViewController: UITableViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        navigationController?.navigationBar.isTranslucent = false
```

Comment:
Do we know the context color & want to use `navigationBar.useContextColor(color)` instead?

Reply:
im not seeing color readily available.